modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/parsers/AbstractGetFeatureParserTest.java

```diff
@@ -486,12 +537,25 @@ public void testParseCubeWerx_RoadSeg() throws Exception {
                 getTypeView(featureName, schemaLocation, CUBEWERX_ROADSEG.CRS, properties);
 
         final GetFeatureParser parser =
-                getParser(featureName, schemaLocation, featureType, CUBEWERX_ROADSEG.DATA);
+                getParser(featureName, schemaLocation, featureType, CUBEWERX_ROADSEG.DATA, null);
 
         int nof = parser.getNumberOfFeatures();
         assertEquals(-1, nof);
 
-        FeatureVisitor assertor = new FeatureAssertor(featureType);
+        final FeatureAssertor assertor = new FeatureAssertor(featureType);
+        List<Geometry> geometries = new ArrayList<Geometry>();
+        geometries.add(
```

Comment:
So here geometries are read in their native order (same as parsed GML file), and the CRS returned is EPSG:4269 with flipped axis order (lat and then lon). Right? Is there also a test for the new axis flipping code in the pull parser?

Reply:
PullParserTest extends this one, so this is actually testing (also) PullParser. Does this answer your question?

Reply:
I think it does not, but not 100% sure. Unless I'm mistaken the ordinates in the GML file are lat/lon, and the tested ordinates are also lat/lon, so where is the new flipping code path being tested?

Reply:
I added two more tests, for the northeast and eastnorth cases, so we should cover all combinations with those.